clients/java/src/main/java/com/google/myanmartools/Transliterate.java

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.myanmartools;
+
+import java.util.ArrayList;
+
+/**
+ * A Java engine for running the phases of transliteration, similar to the ICU4J
+ * transliterator.
+ */
+
+public class Transliterate {
```

Comment:
This is a public class in a public library, which means it is our responsibility to make good API decisions.

Right now the class is mutable.  This is generally no longer considered a good practice.

I suggest making this class publicly immutable.  The easiest way to do this without changing any of your existing code would be to simply make all mutating methods (setters) package-private.

After you do this, add javadoc saying "this class is thread-safe and has immutable behavior."